src/NzbDrone.Core/Indexers/IndexerBase.cs

```diff
@@ -94,7 +92,7 @@ protected virtual IList<ReleaseInfo> CleanupReleases(IEnumerable<ReleaseInfo> re
             result.ForEach(c =>
             {
                 // Use multi languages from setting if ReleaseInfo languages is empty
-                if (c.Languages.Empty() && MultiRegex.IsMatch(c.Title) && settings.MultiLanguages.Any())
+                if (c.Languages.Empty() && FileNameBuilder.MultiRegex.IsMatch(c.Title) && settings.MultiLanguages.Any())
```

Comment:
If this is done in `AggregateLanguages` does it still need to be done here too?

Reply:
Not sure what you mean here.. this function is called from different places, RSS Sync, Interactive searches, release/push